packages/serverless-contracts/src/features/lambdaHandler.ts

```diff
@@ -6,19 +6,48 @@ import {
   EventBridgePayloadType,
   getApiGatewayHandler,
   getEventBridgeHandler,
-  SwarmionApiGatewayHandler,
   SwarmionEventBridgeHandler,
 } from 'contracts';
+import {
+  BodyType,
+  HeadersType,
+  InternalSwarmionApiGatewayHandler,
+  OutputType,
+  PathParametersType,
+  QueryStringParametersType,
+} from 'contracts/apiGateway/types';
+import {
+  ApiGatewayAuthorizerType,
+  ApiGatewayIntegrationType,
+} from 'contracts/apiGateway/types/constants';
 import { ServerlessContract } from 'types';
 
 /**
  * must match the type of getApiGatewayHandler
  */
-export function getHandler<Contract extends ApiGatewayContract>(
+export function getHandler<
```

Comment:
We should have an example in the docs to show how to type extra arguments, now that it is enabled.

Reply:
Done!